librad/tests/working_copy.rs

```diff
@@ -59,7 +58,7 @@ use librad_test::{
 /// 7. peer2 creates an include file, based of the tracked users of the project
 /// i.e. peer1 8. peer2 includes this file in their working copy's config
 /// 9. peer2 fetches in the working copy and sees the commit
-#[tokio::test]
+#[tokio::test(core_threads = 2)]
```

Comment:
Do we know why we have to increase the thread count here?

Reply:
Actually, I don't know. It didn't seem to work on the single-threaded runtime at some point, irrespective of the `block_in_place`, but perhaps that should receive a bit more analysis.

Reply:
Without it, I get this weird error `thread 'can_fetch' panicked at 'can call blocking only when running on the multi-threaded runtime'`

Reply:
Yeah that's expected, `block_in_place` is only available on the multithreaded runtime. The question is if it worksed if we change that to `spawn_blocking`, and remove that `core_threads` thing.

Reply:
Alright, so, it does work if we make `PeerStorage` always use `spawn_blocking`. I'm leaving `block_in_place` in place for tests, because it's just much more convenient to use (you don't have to move it move it). Which means that the tests where it is used need to have `core_threads > 1`.

Reply:
Ah you beat me to it :joy: 

There's only one test that needs it I think: `fetches_on_gossip_notify`

Reply:
Because I came to the same conclusion as you :+1:

Reply:
@kim the working copy tested also worked for me without the core_threads but I see you didn't change it?